app/views/HomePageView.scala.html

```diff
@@ -58,7 +59,16 @@ <h2 class="govuk-heading-m">@messages("homepage.h2.2")</h2>
         <a href="@routes.ProfileController.onPageLoad().url" class="govuk-link">@messages("homepage.p5.linkText")</a>
     </p>
 
-    <h2 class="govuk-heading-m">@messages("homepage.h2.3")</h2>
+    @if(appConfig.downloadFileEnabled) {
+        <h2 class="govuk-heading-m">@messages("homepage.h2.3")</h2>
+        @if(downloadLinkMessagesKey == "homepage.downloadLinkText.noGoodsRecords") {
```

Comment:
AC 2.2 & 2.3 ✅: If user has goods records in their profile display hyperlink, If user does not have goods records in their profile display